libnetdata/ebpf/ebpf.c

```diff
@@ -541,10 +541,12 @@ static netdata_run_mode_t ebpf_select_mode(char *mode)
     return MODE_ENTRY;
 }
 
-void ebpf_update_module_using_config(ebpf_module_t *modules)
+void ebpf_update_module_using_config(ebpf_module_t *modules, int source)
```

Comment:
I suppose it doesn't matter much for this case because the function is only used in 1 place, but I wonder if it should be documented what `source` is expected to be, e.g. `0 == from stock, 1 == from user`, etc.

Reply:
We have some undocumented functions, I plan to bring a PR for this.